examples/forward/point_mass.py

```diff
@@ -0,0 +1,53 @@
+"""
+Point Mass Forward Modelling on Cartesian Coordinates
+=====================================================
+
+The simplest geometry used to compute gravitational fields are point masses.
+Although modelling geologic structures with point masses can be challenging, thery are
+very usefull for other purposes: creating synthetic models, solving inverse problems
+very quickly, generating equivalent sources for interpolation or gridding, etc.
+Besides, the gravitational fields generated by point masses can be quickly computed
+either in Cartesian or in geocentric spherical coordinate system.
+We will compute the downward component of the gravitational acceleration generated by
+a set of point masses on a computation grid given in Cartesian coordinates. We will do
+it throught the :func:`harmonica.point_mass_gravity` function.
+"""
+import harmonica as hm
+import verde as vd
+import matplotlib.pyplot as plt
+import matplotlib.ticker
+
+
+# Define two point masses with oposite mass values of 10000000 kg
+easting = [5e3, 15e3]
+northing = [5e3, 15e3]
+vertical = [-5e3, -2.5e3]
+points = [easting, northing, vertical]
+masses = [10e6, -10e6]
```

Comment:
Now that I read it, "negative mass" sounds terrible :) Maybe we can keep them all positive? The depth difference can be sharper so that we can notice it in the plot.

Reply:
Negative mass is a strange concept for someone coming from Physics (helloo :hand: ), but I think showing a negative mass **contrast** is a great example for Geophysicists. In fact, given that all the surrounding volume has zero mass, it could be understood as these point masses are trying to model anomalous density distributions, and the computed gravitational field is just the anomalous gravitational field. Maybe we could rename `masses` to `mass_contrasts` to make it more clear.

Regarding the depth difference, no problem with that!

Reply:
I think even a "mass contrast" is a bit strange. Things get strange when collapsed to a point. I think we can leave it as "masses" but maybe comment on the example why we're doing negative?